packages/core/addon/templates/chart-tooltips/date.hbs

```diff
@@ -1,13 +1,16 @@
 {{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 <div class="title">{{this.request.logicalTable.timeGrain}} {{this.x}}</div>
-<div class="sub-title">{{metric-format (get this.seriesConfig "metric") this.request.dataSource}}</div>
-<ol class="hover-metrics">
-  {{#each this.tooltipData as |series|}}
-    <li class="metric-val">
-      <svg class="series-indicator">
-        <circle cx="50%" cy="50%" r="40%" class="chart-series-{{series.seriesIndex}}"/>
-      </svg>
-      <span>{{tooltip-value-formatter series.value this.seriesConfig.metric (object-at series.seriesIndex this.rowData)}}</span>
-    </li>
-  {{/each}}
-</ol>
+{{#let (metric-format this.seriesConfig.metric this.request.dataSource) as |metricName|}}
+  <div class="sub-title">{{metricName}}</div>
+  <ol class="hover-metrics">
+    {{#each this.tooltipData as |series|}}
+      <li class="metric-val">
+        <svg class="series-indicator">
+          <circle cx="50%" cy="50%" r="40%" class="chart-series-{{series.seriesIndex}}"/>
+        </svg>
+        <span>wowee</span>
```

Comment:
?
```suggestion
```

Reply:
Yea I put this in for testing, and took it out locally, but didn't push the changes